<?xml version="1.0"?>
<sdf version="1.7">
  <model name="bin_model">
    <!--
    Axes:
      +X - Pointing towards the front (open)
      +Y - Pointing to left side
      +Z - Up
    Origin:
      (0, 0, 0) at the center bottom of the bin
    -->
    <link name="bin_base">
      <inertial>
        <mass>18.70</mass>
        <inertia>
          <ixx>0.79</ixx>
          <ixy>0</ixy>
          <ixz>0</ixz>
          <iyy>0.53</iyy>
          <iyz>0</iyz>
          <izz>1.2</izz>
        </inertia>
      </inertial>
      <visual name="left">
        <pose>0 0.29 0.105 0 0 0</pose>
        <geometry>
          <box>
            <size>0.49 0.05 0.21</size>
          </box>
        </geometry>
        <material>
          <diffuse>0.4 0.4 0.4 1.0</diffuse>
        </material>
      </visual>
      <collision name="left">
        <pose>0 0.29 0.105 0 0 0</pose>
        <geometry>
          <box>
            <size>0.49 0.05 0.21</size>
          </box>
        </geometry>
      </collision>
      <visual name="right">
        <pose>0 -0.29 0.105 0 0 0</pose>
        <geometry>
          <box>
            <size>0.49 0.05 0.21</size>
          </box>
        </geometry>
        <material>
          <diffuse>0.4 0.4 0.4 1.0</diffuse>
        </material>
      </visual>
      <collision name="right">
        <pose>0 -0.29 0.105 0 0 0</pose>
        <geometry>
          <box>
            <size>0.49 0.05 0.21</size>
          </box>
        </geometry>
      </collision>
      <visual name="bottom">
        <pose>0.0 0.0 0.0075 0 0 0</pose>
        <geometry>
          <box>
            <size>0.49 0.63 0.015</size>
          </box>
        </geometry>
        <material>
          <diffuse>0.4 0.4 0.4 1.0</diffuse>
        </material>
      </visual>
      <collision name="bottom">
        <pose>0.0 0.0 0.0075 0 0 0</pose>
        <geometry>
          <box>
            <size>0.49 0.63 0.015</size>
          </box>
        </geometry>
      </collision>
    </link>
  </model>
</sdf>
